src/sentry/monitors/models.py

```diff
@@ -444,21 +452,46 @@ def mark_failed(self, last_checkin=None, reason=MonitorFailure.UNKNOWN):
         if not affected:
             return False
 
-        event_manager = EventManager(
-            {
-                "logentry": {"message": f"Monitor failure: {self.monitor.name} ({reason})"},
-                "contexts": {"monitor": get_monitor_environment_context(self)},
-                "fingerprint": ["monitor", str(self.monitor.guid), reason],
+        failure_occurrence = {
+            "event": {
                 "environment": self.environment.name,
-                # TODO: Both of these values should be get transformed from context to tags
-                # We should understand why that is not happening and remove these when it correctly is
-                "tags": {"monitor.id": str(self.monitor.guid), "monitor.slug": self.monitor.slug},
+                "event_id": uuid.uuid4().hex,
+                "platform": "crons",
+                "project_id": self.monitor.project_id,
+                "received": timezone.now(),
+                "sdk": None,
+                "tags": {
+                    "monitor.id": str(self.monitor.guid),
+                    "monitor.slug": self.monitor.slug,
+                },
+                "timestamp": timezone.now(),
```

Comment:
can be problematic if we want to say last good check-in (and that was > 1 month ago)